app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt

```diff
@@ -107,9 +107,7 @@ fun KaigiApp(
                 drawerSheetContent = {
                     DrawerSheetContent(
                         selectedDrawerItem = kaigiAppScaffoldState.selectedDrawerItem,
-                        onClickDrawerItem = { drawerItem ->
-                            kaigiAppScaffoldState.navigate(drawerItem)
-                        }
+                        onClickDrawerItem = kaigiAppScaffoldState::navigate,
```

Comment:
May I ask if we can fix the problem by adding `Immutable` to KaigiAppScaffoldState? 👀

Reply:
Good idea. Since `KaigiAppScaffoldState` as a `DrawerState` which is `Stable`, should we use `Stable` for the `KaigiAppScaffoldState` instead of `Immutable`?

Reply:
I think so too!
But I think we should have both of codes, method reference and stable. So now, I merge this PR. Thanks

Reply:
> we should have both of codes, method reference and stable

@takahirom Agree. Thanks for merging. If you need any follow-up please assign me to a ticket.